### ISSUE TITLE
convert to HF format

### ISSUE THREAD

**Issue by amirhossein-yousefi** [AnswerDotAI/ModernBERT#209]
could you provide the script that convert the pretrained model with your script to the huggingface format? I can see some one refering to convert_to_hf.py but I can not find it

**Comment by NohTow** [comment]
Hello,

It is [there](https://github.com/AnswerDotAI/ModernBERT/blob/pretraining_documentation/convert_to_hf.py).
We should merge this branch soon.

**Comment by sagawatatsuya** [comment]
Hello.
With the mentioned code, I converted one of modernbert's checkpoints([here](https://huggingface.co/answerdotai/ModernBERT-base-training-checkpoints/blob/main/pretrain/ep0-ba272000-rank0.pt)).
```
python convert_to_hf.py   --output-name modernbert-hf   --output-dir ./output_model   --input-checkpoint ./ep0-ba272000-rank0.pt   --pytorch-bin 
```

After conversion, I executed the following code, but all output logits became nan values.
```python
from transformers import AutoTokenizer, AutoModelForMaskedLM

model_id = "./output_model/modernbert-hf"
tokenizer = AutoTokenizer.from_pretrained(model_id)
model = AutoModelForMaskedLM.from_pretrained(model_id)
text = "The capital of France is [MASK]."
inputs = tokenizer(text, return_tensors="pt")
model = model.to("cuda")
inputs = {k: v.to("cuda") for k, v in inputs.items()}

outputs = model(**inputs)
print(outputs)
```

How can I solve this?
